prisma/seed.ts

```
@@ -1,86 +1,145 @@
+  "Minneapolis, United States",
+  "Toronto, Canada",
+  "Bidi Bidi, Uganda",
+  "Dadaab, Kenya",
+  "Paris, France"
 ];
 
 // Helper function to generate slug from title
-function generateSlug(title: string): string {
-  return title
+function generateSlug(title: string, index: number, usedSlugs: Set<string>): string {
+  let baseSlug = title
```

ESLint

[error] 'baseSlug' is never reassigned. Use 'const' instead.

(prefer-const)